Lib/typing.py

```diff
@@ -1158,14 +1158,20 @@ def _generic_class_getitem(cls, args):
                 f"Parameters to {cls.__name__}[...] must all be unique")
     else:
         # Subscripting a regular Generic subclass.
-        for param in cls.__parameters__:
+        try:
+            parameters = cls.__parameters__
+        except AttributeError:
+            raise TypeError(f"type object '{cls.__qualname__}' has no attribute "
+                            f"'__parameters__'; Maybe you forgot to call "
```

Comment:
```suggestion
                            f"'__parameters__'. Maybe you forgot to call "
```

Either use `; maybe` or `. Maybe`. There is no capitalization after a semi-colon.